frontend/src/views/admin/positions/position-list.js

```diff
@@ -316,7 +316,6 @@ export function ConnectedPositionsList({
         {
             Header: generateHeaderCell("Hours/Assignment"),
             accessor: "hours_per_assignment",
-            maxWidth: 64,
```

Comment:
Why did you take away all these max widths?

Reply:
Many of these columns had very narrow width where the column name or the content itself was hardly visible (half of the word is cut off), so I removed them so they can fit their content.

Reply:
For columns that are displaying a single number, you should keep them skinny. There's no point in making them wide, and it makes the data harder to understand when they're wide.

Reply:
roger! I will make the numeric columns a fixed width then.